docs/topics/mta-web-applying-assessments-to-other-apps.adoc

```diff
@@ -6,7 +6,7 @@
 [id="mta-web-applying-assessments-to-other-apps_{context}"]
 = Applying assessments to other applications
 
-Many applications are similar enough to each other that you might want to apply a completed assessment of one application to another application. This can save time and provide consistent answers to assessment questions for similar applications.
+Many applications are similar enough to each other, and you might want to apply a completed assessment of one application to another application. This can save time and provide consistent answers to assessment questions for similar applications.
```

Comment:
@sbeskin-redhat , Could we include a screenshot of the application details page after a successful assessment ?

Reply:
@nachandr 
The application details page is opened just by clicking the application name in Application inventory, correct?
But why do you want in include it here? This procedure does not require opening the application details page.

Reply:
SB : The application details page is opened just by clicking the application name in Application inventory, correct ?
NC : That's correct .

I was reviewing the preview https://deploy-preview-671--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#assessing-and-analyzing-applications and forgot to specify where this procedure should go.

This needs to go in step 7 of section 5.1. Would it be possible for you to fix that through this PR ?
5.1. Assessing applications 
7. Click Save and Review to view the assessment.

Prior to 6.1, there would be an arrow next to each application. Clicking the arrow would show the application details including Review and Assessment fields.
In 6.1, the arrow was removed. So, I thought a screenshot would be helpful.

Reply:
@nachandr 
Sorry, I missed your last comment. 
I am not sure I understand what you mean. Which procedure needs to be in 7 of section 5.1?

Reply:
@sbeskin-redhat , 

Do let me know if you have any questions for me.

Current documentation
-----------------------------------------
**5.1. Assessing applications**
7. Click Save and Review to view the assessment.

I would like this to be updated to
-------------------------------------------
**5.1. Assessing applications**
7. Click Save and Review to view the assessment.

The fields from Assessment are now populated on the Application details page.